### ISSUE TITLE
After Redirect cypress is moving to /__/ 

### ISSUE THREAD

**Issue by Giszter** [cypress-io/cypress#9261]
Hi, I have an issue as bellow:
After add username, password and click on Login button I see a page with error `"Whoops there is no page to run"`
Preconditions:

Mac OsCatalina
VPN Cisco AnyConnect

On terminal I see something like below:
```
POST /VM/Account/Login 302 0.343 ms - 52
GET /__/ 200 0.569 ms - -

```
config JSON

```
{
  "baseUrl": "http://vm.dev.q88.com",
  "chromeWebSecurity": false
}
```

**Comment by chenfan-fchen** [comment]
Cypress didn't support redirect directly, you need use GoogleSocialLogin plugin to get the cookie info, then set the needed cookie back to Cypress

**Comment by psanlorenzo** [comment]
@Giszter did you find what was causing you this redirect? It happens to me as well.

**Comment by jennifer-shehane** [comment]
We believe a fix for this was released in [`7.3.0`](https://on.cypress.io/changelog#7-3-0) as part of https://github.com/cypress-io/cypress/pull/15995. Please upgrade to the new version.

If you're still experiencing an issue after upgrade, open a new issue with a fully reproducible example of the issue that we can run.
